frontend/src/pages/ProfilePage/ProfilePage.tsx

```diff
@@ -0,0 +1,56 @@
+import BoxOne from "../../components/ProfileBoxes/BoxOne";
+
+import classes from "./ProfilePage.module.scss";
+import { useState } from "react";
+
+const ProfilePage = () => {
+  const [username, setUsername] = useState(""); // To store the username
```

Comment:
I think to make life easier you could've made the entire user response one state